features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx

```diff
@@ -309,80 +304,90 @@ class APICreateGraphQL extends React.Component {
-                    <div>
-                        <div>
+                                    return (
+                                        <Step key={label} {...props}>
+                                            <StepLabel {...labelProps}>{label}</StepLabel>
+                                        </Step>
+                                    );
+                                })}
+                            </Stepper>
                             <div>
-                                <Button
-                                    disabled={activeStep === 0}
-                                    onClick={this.handleBack}
-                                    className={classes.button}
-                                >
-                                Back
-                                </Button>
-                                <Button
-                                    variant='contained'
-                                    color='primary'
-                                    onClick={this.handleNext}
-                                    className={classes.button}
-                                    disabled={
-                                        (valid.graphQLFile.invalidFile && uploadMethod === 'file')
-                                    }
-                                >
-                                    {activeStep === steps.length - 1 ? (
-                                        'Finish'
-                                    ) : (
-                                        <FormattedMessage
-                                            id='Apis.GraphQL.APICreateGraphQL.next'
-                                            defaultMessage='Next'
+                                {activeStep === 0 && (
+                                    <ProvideGraphQL
+                                        {...provideGraphQLProps}
+                                    />
+                                )}
+                                {activeStep === 1 && (
+                                    <React.Fragment>
+                                        <APIInputForm
+                                            api={api}
+                                            handleInputChange={this.updateApiInputs}
+                                            valid={valid}
                                         />
-                                    )}
-                                </Button>
+                                    </React.Fragment>
+                                )}
+                            </div>
+                            <div>
+                                <div>
+                                    <div>
+                                        <Button
+                                            disabled={activeStep === 0}
+                                            onClick={this.handleBack}
+                                            className={classes.button}
+                                        >Back
```

Comment:
Need to localize the string

Reply:
This file is no longer used as the code had been refactored with react hooks 
https://github.com/wso2/carbon-apimgt/pull/6944/files
Therefore, we don't want to add localization at here